server/api/utils/scheduler.py

```diff
@@ -94,7 +94,6 @@ def _append_access_key_secret_to_labels(self, labels, secret_name):
         if secret_name:
             labels = labels or {}
             labels[self._db_record_auth_label] = secret_name
-        return labels
```

Comment:
if labels is `None` this will not work. the labels attr in the inner function scope will point to a new dict but the outer function's labels will still be `None`